app/src/main/java/com/fluffybros/tujefe/MainViewModel.kt

```diff
@@ -6,9 +6,7 @@ import androidx.lifecycle.ViewModel
 import com.fluffybros.tujefe.ui.home.HomeRecyclerItem
 
 class MainViewModel : ViewModel() {
-    private val _homeList = MutableLiveData<List<HomeRecyclerItem>>().apply {
-        value = ArrayList()
-    }
+    private val _homeList = MutableLiveData<List<HomeRecyclerItem>>(ArrayList())
```

Comment:
let's use `listOf()` instead of `ArrayList()`

Reply:
i don't see this change??

Reply:
switch to [using listOf](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/list-of.html) instead of `ArrayList()`